src/main/scala/com/databricks/spark/xml/util/TypeCast.scala

```diff
@@ -112,15 +113,23 @@ private[xml] object TypeCast {
     // 2002-05-30T21:46:54+06:00
     DateTimeFormatter.ISO_OFFSET_DATE_TIME,
     // 2002-05-30T21:46:54.1234Z
-    DateTimeFormatter.ISO_INSTANT
+    DateTimeFormatter.ISO_INSTANT.withZone(ZoneId.of("UTC"))
```

Comment:
This should already be UTC; I don't think we want to change this
https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT

Reply:
I haven't been able to run `Timestamp.from(ZonedDateTime.parse(value, DateTimeFormatter.ISO_INSTANT).toInstant)`. The doc also says:

> As such, an Instant cannot be formatted as a date or time without providing some form of time-zone.

I feel like that ISO_INSTANT should not be in `supportedXmlTimestampFormatters`.
Same issue here: https://stackoverflow.com/questions/25612129/java-8-datetimeformatter-and-iso-instant-issues-with-zoneddatetime

Reply:
I think we need to support the format; I think I copied these from a list of standard formats according to XSD specs or something. It's super standard.

We're parsing rather than formatting here. What's the issue - does it not work with ZonedDateTime? maybe never did, if so.

This would probably also be better if we somehow cached the parsing of these built-in patterns as parser/formatters, but that can happen separately with some optimization.

Reply:
Hm OK I see why you made that change, it doesn't seem to think it's "UTC" otherwise, when it should be by nature. OK leave it in

Reply:
OK different idea - what if we write the parsing without ZonedDateTime? `Timestamp.from(Instant.from(format.parse(value)))` Does that help? seems to be simpler, not sure why it wasn't written that way in the first place

Reply:
No, ISO_INSTANT does not work with ZonedDateTime